be/src/exec/schema_scanner/schema_be_cloud_native_compactions_scanner.cpp

```diff
@@ -40,7 +40,13 @@ SchemaScanner::ColumnDesc SchemaBeCloudNativeCompactionsScanner::_s_columns[] =
         {"START_TIME", TYPE_DATETIME, sizeof(DateTimeValue), true},
         {"FINISH_TIME", TYPE_DATETIME, sizeof(DateTimeValue), true},
         {"PROGRESS", TYPE_INT, sizeof(int32_t), false},
-        {"STATUS", TYPE_VARCHAR, sizeof(StringValue), false}};
+        {"STATUS", TYPE_VARCHAR, sizeof(StringValue), false},
+        {"READER_IO_MS", TYPE_BIGINT, sizeof(int64_t), false},
```

Comment:
if ms_remote + ms_local_disk == ms, can we remove READER_IO_MS?

Reply:
> if ms_remote + ms_local_disk == ms, can we remove READER_IO_MS?

In current implementation, `ms_remote + ms_local_disk` were  not expected to equal `io_ms`.